src/components/Modules/Home/HomeContentModule.tsx

```diff
@@ -1,7 +1,29 @@
+import { DarkLightPreviewComponent } from "~/components/Components/DarkLightPreviewComponent/DarkLightPreviewComponent";
+
 type Props = {
-  title: string;
+  darkLightProps: Array<{
```

Comment:
Do not re-type alias Component's type
use `React.ComponentProps<typeof {ComponentName}>`

check Convention Docs [30-1](https://coal-blouse-907.notion.site/30-1-Component-Prop-Type-Alias-Convention-9c8077bb13dc4b40a112ffbe9e63aa93?pvs=4)